airbyte-integrations/connectors/source-rippling/source_rippling/schemas/company_leave_types.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "properties": {
```

Comment:
Can you set additionalProperties to true for each of the schema files? https://docs.airbyte.com/connector-development/schema-reference/